### PR TITLE
add api listBrowsers

### PR DESCRIPTION
**Pull request by milahu**
solve issue #3919

code is not tested

small change in CLI: removed double quotes from output

### CODE REVIEW

File: src/api/exportable-lib/index.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ const SelectorBuilder       = lazyRequire('../../client-functions/selectors/sele
 const role                  = lazyRequire('../../role');
 const createRequestLogger   = lazyRequire('../request-hooks/request-logger');
 const createRequestMock     = lazyRequire('../request-hooks/request-mock');
+const browserProviderPool   = lazyRequire('../../browser/provider/pool');
```
Comment (AndreyBelym) [inline src/api/exportable-lib/index.js:7]: `exportable-lib` should contain only Test API. Please extract `list-browsers` to the `/src/utils/list-browsers` and export it through [/src/embedding-utils](https://github.com/DevExpress/testcafe/blob/master/src/embedding-utils.js)

File: src/cli/cli.js [hunk 2]
```diff
@@ -124,25 +120,17 @@ async function runTests (argParser) {
...
-        await browserProviderPool.dispose();
-
-        if (providerName === 'locally-installed')
+    try {
+        const browserNames = await createTestCafe.listBrowsers(providerName);
+        if (providerName == 'locally-installed')
             console.log(browserNames.join('\n'));
         else
-            console.log(browserNames.map(browserName => `"${providerName}:${browserName}"`).join('\n'));
+            console.log(browserNames.map(n => providerName+':'+n).join('\n'));
```
Comment (AndreyBelym) [inline src/cli/cli.js:128]: Could you please keep `browserName` as the parameter name in the arrow function?

**Review by AndreyBelym — COMMENTED** [review]
Thank you for your contribution. Please run `npx gulp lint` in your local copy and fix linting errors. At least a single unit test ([/test/server](https://github.com/DevExpress/testcafe/tree/master/test/server)) is required for any new public API.

**Comment by milahu** [comment]
closing in favor of PR #5521

**Pull request closed without merging**